### PR TITLE
Css fix for add ons (extensions) detail page

### LINKED ISSUE #4878: The add-on and author's name are too close
STR:
1. Submit an add-on that contains "y" "q" "p" "g" in it's name
2. Go to the add-ons detail page
3. Observe the Add-on details section

Expected result:
It's a reasonable space between the Add-on's Name and Author.

Actual result:
The two names are too close, they look jammed in.

Notes:
This issue is reproducible on AMO dev and AMO stage with FF59 (Win10).

I'll give below two examples to make it more clear
too close

and another one 
another example

### PR DESCRIPTION
**Pull request by biskit1**
Fixes #4878 

This PR introduces css changes to the details page of individual add ons. 

The original issue was opened to fix the line containing the author, which at times would overlap with the title (depending on the letters contained in the title, such as `g, y, p, q` and the size of screen) like so: 
<p align="center">
  
</p><br/>

To fix this problem, I just added a `line-height` of `1.5` to the `Addon-author` class:
<p align="center">
  
</p><br/>

While implementing this change and testing the results, I also noticed that at times the title itself ran into the same problem: 
<p align="center">
  
</p>
and another example: 
<p align="center">
  
</p><br/>

So I added a `line-height` property to the `Addon-title` class as well, with a value of `normal` (1.5 was too much for the title): 
<p align="center">
  
</p><br/>

There were no tests found for the files I modified, so I ran `yarn lint` and corrected any formatting errors.

### CODE REVIEW

**Comment by rebmullin** [comment]
Thanks for this @biskit1 : ) Regarding your changes, I still show your line height for `Addon-title` as 1.5 tho.. per your comment above it seems like you meant to change that..?  

I agree that the title's line height could be improved a bit (in case there are 2 lines) but I think `normal` might be a bit too much - what do you think about something a little smaller (but bigger than what it currently is at :)?

**Comment by biskit1** [comment]
Thanks for the feedback @rebmullin! How is `1.3` for `Addon-author`? 
I don't know why I automatically jumped to .5, but I agree it was a bit much.

**Comment by rebmullin** [comment]
that’s a really good point @biskit1 - that we need to consider longer usernames. I am not sure how common those are but it still might be a (edge) case to consider.  Regarding the padding…I think that might be a good solution to try out..we did increase the parent line height to 1.1 and since `Addon-author` will inherit that maybe this will look okay with just the extra padding. Do you want to try that and see it how looks at the different breakpoints?

**Comment by biskit1** [comment]
@willdurand yes. I drafted a response a while ago and I’ve been meaning to post and update the PR. will post this afternoon. Apologies for the delay. 
Thanks for following up on this!

**Comment by biskit1** [comment]
@rebmullin I think it's good with the `padding-top: 3px` for `Addon-author`. 
A couple of breakpoints as an example: 

image

image

Another option is to **just** have `line-height: normal` on the `Addon-title`. I found it nicely spaced the title and author line at the various breakpoints, but I think you might find it to be a little bit too for when the title line wraps: 

image

File: src/amo/components/Addon/styles.scss [hunk 1]
```diff
@@ -85,6 +86,7 @@
   @include font-light();
 
   display: block;
+  padding-top: 3px;
```
Comment (willdurand) [inline src/amo/components/Addon/styles.scss]: what is this padding for?

**Pull request merged**